### PR TITLE
CSPG-36014 Support download for stage containers

### PR DESCRIPTION
**Pull request by skasianov**
When used in testing environment we want to run falcon-container-sensor-pull.sh not only against release but stage repository as well

### CODE REVIEW

File: bash/containers/falcon-container-sensor-pull/README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Required Flags:
 Optional Flags:
     -f, --cid <FALCON_CID>                         Falcon Customer ID
     -r, --region <FALCON_CLOUD>                    Falcon Cloud Region [us-1|us-2|eu-1|us-gov-1] (Default: us-1)
+    -g, --stage <FALCON_STAGE>                     Falcon Build Stage [release|stage] (Default: release)
```
Comment (carlosmmatos) [inline bash/containers/falcon-container-sensor-pull/README.md:73]:
```suggestion
    -b, --build-stage <FALCON_STAGE>                     Falcon Build Stage [release|stage] (Default: release)
```

Comment (carlosmmatos) [reply bash/containers/falcon-container-sensor-pull/README.md:73]: Can we do this instead of -g, —stage?

Comment (skasianov) [reply bash/containers/falcon-container-sensor-pull/README.md]: Sure. Updated the PR

**Comment by carlosmmatos** [comment]
@skasianov just to be clear, is this for internal purposes only?

**Comment by skasianov** [comment]
Yes. 
Original requirement is for MTESTING project https://jira.cs.sys/browse/CSPG-35374
used by https://jenkins-cspm.cicd.dc/job/CSPM/job/CSPG-MTESTING/ pipeline internally.

**Comment by carlosmmatos** [comment]
@skasianov I removed the refs to our internal stuff.. 

In the meantime, this is on hold because we are trying to figure out the best way to proceed with this since this is really only an internal request.

**Review by evanstoner — APPROVED** [review]

**Pull request merged**